docs/release_notes/v1.9.5.md

```diff
@@ -0,0 +1,19 @@
+# Dapr 1.9.5
+
+### Fixes an issue in the CosmosDB state store component when performing transaction request with ETags
+
+#### Problem
+
+When setting any ETag in a transaction request inside CosmosDB, dapr exits due to a nil pointer dereference.
+
+### Impact
+
+This issue impacts all users of CosmosDB state store.
```

Comment:
"...all users of the Azure Cosmos DB state store."

Reply:
Also do you know when this issue started? 1.9? If so, can you add the versions that are impacted, such as 1.x-1.9.4

Reply:
presumably 1.9.0-1.9.4 since we didn't use this SDK yet in 1.8.X